modules/op-webstat-pipeline.adoc

```diff
@@ -0,0 +1,25 @@
+// This module is included in the following assemblies:
+// * create/working-with-pipelines-web-console.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="op-webstat-pipeline_{context}"]
+= Viewing the statistics for a specific pipeline
+
+You can view statistic information related to a particular pipeline.
+
+.Prerequisites
+
+* You installed the {pipelines-shortname} web console plugin.
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, select *Pipelines* -> *Pipelines*.
+
+. Click a pipeline in the list of pipelines.
++
+A statistics overview displays. This overview includes a graph reflecting the number and status of pipeline runs for this pipeline over a time period, as well as the total, average, and maximum durations of pipeline execution and total number of pipeline runs over the same period.
```

Comment:
After clicking on the Pipeline in the Pipelines list page, a user needs to click on the `Metrics` tab on the Pipeline details to see the statistic information of a Pipeline.